index-zh-cn.bs

```diff
@@ -0,0 +1,355 @@
+<pre class="metadata">
+Title: Accelerated Shape Detection in Images
+Repository: wicg/shape-detection-api
+Status: ED
+ED: https://wicg.github.io/shape-detection-api
+Shortname: shape-detection-api
+Level: 1
+Editor: Miguel Casas-Sanchez, w3cid 82825, Google Inc., mcasas@google.com
+Abstract: 本文档描述了一套Chrome中针对静态和/或动态图像的图形识别（如：人脸识别）API。
```

Comment:
Add here a line:
`Translation: en-US https://wicg.github.io/shape-detection-api`

and to the index.bs, add a line:
`Translation: zh-CN https://wicg.github.io/shape-detection-api/index-zh-cn.html`

And don't forget compiling both by using the appropriate bikeshed service, e.g.
`curl https://api.csswg.org/bikeshed/ -F file=@index.bs > index.html`